packages/eslint-config-typescript/index.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  extends: ["@madebywild/eslint-config-base"],
+  overrides: [
+    {
+      files: ["**/*.js?(x)"],
+      rules: {},
+    },
+    {
+      files: ["**/*.ts?(x)"],
+      rules: {
+        "@typescript-eslint/no-explicit-any": "warn",
+        "@typescript-eslint/no-empty-function": "warn",
+        "@typescript-eslint/no-unused-vars": [
+          "warn",
+          {
+            ignoreRestSiblings: true,
+            args: "none",
+            varsIgnorePattern: "^_",
+          },
+        ],
+      },
+    },
```

Comment:
We can move this to the top-level `rules`object now I think.

Reply:
@r0skar I wasn't sure if that would also interfere with js files in a typescript project, but I guess that shouldn't happen anyway (or could be me just remembering less nice linting from a couple years ago)

Reply:
Oh, good point. I didnt think about that. You are right that the rules would apply to JS files. On one hand I think some of the rules (eg. no-empty-function) can be useful for JS files as well. But still, its probably a good idea to keep the override.